samples/AllControls/AllControls/AllControls.fsproj

```
@@ -8,6 +8,7 @@
       <EmbeddedResource Include="Baboon_Serengeti.jpg" />
       <EmbeddedResource Include="styles.css" />
       <PackageReference Include="FSharp.Core" />
+      <PackageReference Include="FSharp.Data" Version="3.0.0" />
```

Versioning is centralized in the `Directory.Build.props` file at the root of the repository.
In this file, you only need to include the package (without version)
Then in the `Directory.Build.props`, you can add the version, just like the other packages.
```suggestion
      <PackageReference Include="FSharp.Data" />
```